content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+  with the files sent to the |DGI| testing environment through Uruware.
+- :guilabel:`Demo`: files are created and accepted automatically in demo mode but are **not** sent
+  to the |DGI|. For this reason, rejection errors will not appear in this mode. Every internal
+  validation can be tested in demo mode. Avoid selecting this option in a production database.
+
+Then, enter the :guilabel:`Uruware Credentials`:
+
+- :guilabel:`Uruware WS Password`
+- :guilabel:`Commerce Code`
+- :guilabel:`Terminal Code`
+
+.. image:: uruguay/electronic-invoice-data.png
+   :alt: Required information for electronic invoice.
+   :align: center
+
+.. note::
+   These credentials can be obtained from the email received when the :ref:`Uruguay account was
+   created <l10n_uy/uruware-account>`.
+
+Configure master data
+---------------------
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+The :doc:`chart of accounts <../accounting/get_started/chart_of_accounts>` is installed by default
+as part of the set of data included in the localization module, the accounts are mapped
+automatically in taxes, default accounts payable, and default accounts receivable.
+
+Accounts can by added or deleted according to the company's needs.
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+Contacts
+~~~~~~~~
+
+To create a contact, navigate to :menuselection:`Contacts app` and select :guilabel:`New`. Then
+enter the following information:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`: required to confirm an electronic invoice.
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`: required to confirm an electronic invoice.
+
+- :guilabel:`Identification Number`:
+
+  - :guilabel:`Type`: select a identification number type.
+  - :guilabel:`Number`: required to confirm an electronic invoice.
+
+Review taxes
+~~~~~~~~~~~~
+
+As part of the Uruguay localization module, taxes are automatically created with its configuration
+and related financial accounts.
+
+.. image:: uruguay/taxes.png
+   :align: center
+   :alt: Taxes for Uruguay.
+
+Review document types
+~~~~~~~~~~~~~~~~~~~~~
+
+Some accounting transactions like *Customer Invoices* and *Vendor Bills* are classified by document
+types. These are defined by the government fiscal authorities, in this case by the |DGI|.
+
+Each document type can have a unique sequence per journal where it is assigned. The data is created
+automatically when the localization module is installed, and the information required for the
+document types is included by default.
+
+.. image:: uruguay/document-types.png
+   :align: center
+   :alt: Document types for Uruguay,
+
+Sales journals
+~~~~~~~~~~~~~~
+
+To generate and confirm an electronic document that will be validated by |DGI|, the sales journal
+needs to be configured with the following:
+
+- :guilabel:`Invoicing Type`: manual invoices are for open invoices previously stamped in another
```

Comment:
The electronic option is missing here with its explanation. And electronic will be set by default.

Reply:
Add following bullet point @samueljlieber

- :guilabel:`Ecuadorina Edi format`: This checkbox will be active by default. This is necessary to send electronic documents via web service to the Ecuadorian government.

Reply:
> Add following bullet point @samueljlieber
> 
> * :guilabel:`Ecuadorian edi format`: This checkbox will be active by default. This is necessary to send electronic documents via web service to the Ecuadorian government.

@rps-odoo Ecuadorian? 🤔

Reply:
@fvz-odoo don't ask me, ask Odoo
![Screenshot 2024-07-08 at 2 31 32 PM](https://github.com/odoo/documentation/assets/79879654/57ccf18a-1fa4-4b20-a794-5b7757e78e37)

Reply:
> @fvz-odoo don't ask me, ask Odoo. I will change it to Spanish.
> 
> ![Screenshot 2024-07-08 at 2 31 32 PM](https://private-user-images.githubusercontent.com/79879654/346716551-57ccf18a-1fa4-4b20-a794-5b7757e78e37.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0NzI3ODQsIm5iZiI6MTcyMDQ3MjQ4NCwicGF0aCI6Ii83OTg3OTY1NC8zNDY3MTY1NTEtNTdjY2YxOGEtMWZhNC00YjIwLWE3OTQtNWI3NzU3ZTc4ZTM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDIxMDEyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk1YjBlNThhYzg5ODAwNzMzZGI2Y2FkMzVlOWY2Y2E4YTRiZjczZjkyYmQ3ODAyNWUwYjUzZjJmMzI2ZmQ3YmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iqixzT38eeGnn2M2y_J_A8UEXxADnghutIdT07yogqo)

Hello @rps-odoo 

Mmmm, I don't understand why we have this setting in the Uruguayan localization. We're probably talking about different things? 

In the Uruguayan localization we have a selection field (l10n_uy_edi_type):
![Screenshot 2024-07-08 at 2 08 27 PM](https://github.com/odoo/documentation/assets/75649179/f4e36d99-052f-44d8-80e5-28feff7b43e3)

If, for some reason, we have that Ecuadorian field in our module, we have something wrong, but I don't see it. Can you please confirm we have it and that we're talking about the same thing?

Let me know.

Cheers!

Fernando

@samueljlieber

Reply:
@fvz-odoo Ops, I am confusing myself. I am also working on another PR for Ecuador and got confused. Don't worry,  everything seems as you explain. @samueljlieber please ignore all these comments and just add the following:

:guilabel:'Invoicing Type': This option will have 'Electronic' value set by default. This is necessary to send electronic documents via web service to the Ecuadorian government.

Reply:
> @fvz-odoo Ops, I am confusing myself. I am also working on another PR for Ecuador and got confused. Don't worry, everything seems as you explain. @samueljlieber please ignore all these comments and just add the following:
> 
> * :guilabel:'Invoicing Type': This option will have 'Electronic' value set by default. This is necessary to send electronic documents via web service to the Ecuadorian government.

One thing @samueljlieber, please, just make sure it says **Uruguayan** government at the end 😅.